src/ToDo.UI/Views/Dialogs/RenameListFlyout.xaml

```diff
@@ -1,50 +1,54 @@
-
-	<utu:AutoLayout Spacing="20"
-					Padding="20,20,20,20"
-					CornerRadius="8"
-					Background="{ThemeResource MaterialSurfaceBrush}">
-
-		<TextBlock Foreground="{ThemeResource MaterialOnSurfaceBrush}"
-				   TextWrapping="Wrap"
-				   Text="Rename list"
-				   Style="{StaticResource MaterialHeadline6}" />
-
-		<TextBox x:Name="ListNameText"
-				 PlaceholderText="Rename list"
-				 Text="{Binding ListName, Mode=TwoWay}"
-				 Style="{StaticResource MaterialFilledTextBoxStyle}" />
-
-		<utu:AutoLayout Spacing="10"
-						Orientation="Horizontal">
-
-			<utu:AutoLayout Spacing="10"
-							Orientation="Horizontal"
-							utu:AutoLayout.CounterAlignment="Start"
-							utu:AutoLayout.PrimaryAlignment="Stretch" />
-
-			<Button Content="Cancel"
-					uen:Navigation.Request="-"
-					utu:AutoLayout.CounterAlignment="Start"
-					Style="{StaticResource MaterialTextButtonStyle}" />
-
-			<Button Content="Rename"
-					uen:Navigation.Request="-"
-					uen:Navigation.Data="{Binding Text, ElementName=ListNameText}"
-					utu:AutoLayout.CounterAlignment="Start"
-					Style="{StaticResource MaterialContainedButtonStyle}" />
-
-		</utu:AutoLayout>
-
-	</utu:AutoLayout>
+﻿<Page
+    x:Class="ToDo.Views.Dialogs.RenameListFlyout"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:ToDo.Views.Dialogs"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:uen="using:Uno.Extensions.Navigation.UI"
+    xmlns:um="using:Uno.Material"
+    xmlns:utu="using:Uno.Toolkit.UI"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+
+    <utu:AutoLayout
+        Padding="20,20,20,20"
+        Background="{ThemeResource MaterialSurfaceBrush}"
+        CornerRadius="8"
+        Spacing="20">
+
+        <TextBlock
+            Foreground="{ThemeResource MaterialOnSurfaceBrush}"
+            Style="{StaticResource MaterialHeadline6}"
+            Text="Rename list"
+            TextWrapping="Wrap" />
+
+        <TextBox
+            x:Name="ListNameText"
+            PlaceholderText="Rename list"
+            Style="{StaticResource MaterialFilledTextBoxStyle}"
+            Text="{Binding ListName, Mode=TwoWay}" />
+
+        <utu:AutoLayout Orientation="Horizontal" Spacing="10">
+
+            <utu:AutoLayout
+                utu:AutoLayout.CounterAlignment="Start"
+                utu:AutoLayout.PrimaryAlignment="Stretch"
+                Orientation="Horizontal"
+                Spacing="10" />
+
+            <Button
+                uen:Navigation.Request="-"
+                Content="Cancel"
+                Style="{StaticResource MaterialTextButtonStyle}" />
+
+            <Button
+                utu:AutoLayout.CounterAlignment="Start"
+                Command="{Binding RenameCommand}"
```

Comment:
There is no need for the RenameCommand anymore normally as it's done by the extension with the navigation normally
```
uen:Navigation.Request="-"
uen:Navigation.Data="{Binding Text, ElementName=ListNameText}"
```

Reply:
Okay, thanks